Spanner/tests/System/PgWriteTest.php

```diff
@@ -53,6 +53,7 @@ public static function setUpBeforeClass(): void
                 bytesfield bytea,
                 datefield date,
                 floatfield float,
+                float4field float4,
```

Comment:
why don't we name it `float32field` instead of `float4field`?

Reply:
Because in PG dialect, 4 bytes float is called `float4`